### DISCUSSION TITLE
Symmetry Mismatch Warning?

### DISCUSSION THREAD

**Discussion by romydhan** [mtex-toolbox/mtex#1987] [Ask Anything]
I get a Warning about a symmetry mismatch, but I can't figure out why, nor if it matters.  Help?

Here is the code to replicate:

    load("variables.mat")
    phase = 'Clinoenstatite'
    cs = ebsd(phase).CS;
    odf_phase = calcDensity(grains(phase).meanOrientation,'halfwidth',8*degree);
    
    f = figure;
    plotPDF(odf_phase,Miller(1,0,0,cs,'hkl'),'upper','uer','projection','eangle','complete','contourf','minmax','antipodal','colorrange',[0,5]);

Thanks, 
Romy Hanna
Asst. Research Professor UT Austin

[variables.mat.zip](https://github.com/mtex-toolbox/mtex/files/13208445/variables.mat.zip)

**Comment by lfgmorales** [comment]
Hi

Well, crystallographically it matters (otherwise there was no reason for the warning :-D) - most important is the fact that you seem to have only one grain of clinoenstatite, and you calculate an ODF of that, which I suspect is not statistically very robust…however, you have like 96% of non indexed points, is that really the case?

cheers


Luiz



On 30 Oct 2023, at 19:16, romydhan ***@***.******@***.***>> wrote:


phase = 'Clinoenstatite'
cs = ebsd(phase).CS;
odf_phase = calcDensity(grains(phase).meanOrientation,'halfwidth',8*degree);

f = figure;
plotPDF(odf_phase,Miller(1,0,0,cs,'hkl'),'upper','uer','projection','eangle','complete','contourf','minmax','antipodal','colorrange',[0,5])

**Reply by romydhan** [reply]
Thanks Luiz.  Yes, I realize that calculating an ODF from a single grain is silly, this is just more of an exercise to test my code.  I loop through all the phases present, although in the actual code that this is pulled from I set a lower limit to the number of grains required to run this part of it (so in actuality it would never be run).  But, it was unexpected to me for this warning to pop up.  Are you saying it's only occurring because there is one grain?  That's not what the warning implies....

Yes, I have a very high amount of misindexed points because these are small, isolated grains in a largely amorphous matrix (it's a primitive carbonaceous chondrite).

**Reply by romydhan** [reply]
Also, a little more context, if you actually run the code this is what the warning looks like:

**Warning: Symmetry missmatch! The following crystal frames seem to be different

  Clinoenstatite (12/m1, X||a*, Y||b*, Z||c)
  Clinoenstatite (12/m1, X||a*, Y||b*, Z||c)**
 
So, to me those look the same?  Thus my question - why I am I getting this warning?  This is the only phase it happens with.  Perhaps it is some weird thing popping up because there's only one grain?  But then a warning about symmetry mismatch should not occur I would think.

Romy

**Comment by lfgmorales** [comment]
Hi,

No, this is not what the warning implies, and I believe is because your beta angle is 71. something, while it should be >90° to my understanding…accordingly to the literature, around 108.3°…so I beliebe the warning comes from there.

cheers

Luiz


On 30 Oct 2023, at 20:09, Romy Hanna ***@***.******@***.***>> wrote:



Thanks Luiz. Yes, I realize that calculating an ODF from a single grain is silly, this is just more of an exercise to test my code. I loop through all the phases present, although in the actual code that this is pulled from I set a lower limit to the number of grains required to run this part of it (so in actuality it would never be run). But, it was unexpected to me for this warning to pop up. Are you saying it's only occurring because there is one grain? That's not what the warning implies....

Yes, I have a very high amount of misindexed points because these are small, isolated grains in a largely amorphous matrix (it's a primitive carbonaceous chondrite).

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1987#discussioncomment-7427608>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN2IJ7H4AUEWAQHD4VDYB73NPAVCNFSM6AAAAAA6WOUDIGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TIMRXGYYDQ>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1987#discussioncomment-7427608", "url": "https://github.com/mtex-toolbox/mtex/discussions/1987#discussioncomment-7427608", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by romydhan** [reply]
Excellent, this helps.  Thanks Luiz!
